position_joint_trajectory_controller:
  constraints:
    goal_time: 5.0
    stopped_velocity_tolerance: 0